js/src/setup-ads/ads-stepper/index.js

```diff
@@ -55,21 +62,38 @@ const AdsStepper = ( { formProps } ) => {
 	};
 
 	const handleSetupAccountsContinue = () => {
-		continueStep( '2' );
+		continueStep( hasGoogleAdsConnection ? '1' : '2' );
```

Comment:
```suggestion
		continueStep( hasGoogleAdsConnection '2' );
```
This will always be `2` since the accounts step will always be displayed as the first step.

Reply:
Fixed.